lib/devtools-require.js

```diff
@@ -15,8 +15,8 @@ const ID_RENAMES = [
     replacement: "devtools/client/"
   },
   {
-    regex: /^devtools\/shared\/styleinspector\//,
-    replacement: "devtools/styleinspector/"
+    regex: /^devtools\/shared\/inspector\//,
+    replacement: "devtools/inspector/"
```

Comment:
Actually, we may need to tweak the module a bit more, since it currently runs *all* replacements first, and then tries to load the module. To support more than one old version, we'll need to do something like attempt requiring after *each* replacement.